http/src/main/java/com/redhat/lightblue/client/http/LightblueHttpClient.java

```diff
@@ -206,7 +205,7 @@ public Locking getLocking(String domain) {
      * @see com.redhat.lightblue.client.LightblueClient#metadata(com.redhat.lightblue .client.request.LightblueRequest)
      */
     @Override
-    public DefaultLightblueMetadataResponse metadata(LightblueRequest lightblueRequest) throws LightblueException {
+    public DefaultLightblueMetadataResponse metadata(LightblueRequest lightblueRequest) throws LightblueParseException, LightblueResponseException, LightblueHttpClientException {
```

Comment:
The client side try/catch block can still catch the generic LightblueException. Why not provide the ability to handle individually if desired?